docs/secondary-setup.md

```diff
@@ -0,0 +1,6 @@
+# Secondary setup
```

Comment:
Lets remove this file I don't think this section adds anything that justifies it's own file.

We aren't describing any other jobs that are run and the Secondary setup probably refers to the docker-compose setup which is just weird to have here.